src/screens/Watch/Utils/index.js

```diff
@@ -82,9 +84,11 @@ export const getVideoStyle = (options) => {
   const {
     brightness = SCREEN_OPACITY_100,
     contrast = 1,
+    rotateColor = ROTATE_COLOR_0,
+    invert = INVERT_0,
   } = options;
   const videoStyle = {
-    filter: `brightness(${brightness}) contrast(${contrast})`,
+    filter: `brightness(${brightness}) contrast(${contrast}) hue-rotate(${rotateColor}) invert(${invert})`,
```

Comment:
For performance, please remove a filter from the filter chain if it is a no-op. e.g.
```
filter=''
if (Math.abs(brightness - SCREEN_OPACITY_100)>0.01) filter += `brightness(${brightness}) `
if (Math.abs(contrast - 1) > 0.01) filter += `contrast(${contrast}) `
if (rotateColor != ROTATE_COLOR_0) filter += `hue-rotate(${rotateColor}) `
if( invert != INVERT_0) filter += `invert(${invert}) `
```
(